fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/PIDs.java

```diff
@@ -43,6 +44,8 @@
 
     private static final Logger log = LoggerFactory.getLogger(PIDs.class);
 
+    //[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}
```

Comment:
Is this needed for documentation purposes?

Reply:
Nope, forgot to delete that while I was still deciding where to incorporate it. Removed it.